File: python/2344.py
n = int(input())

if n < 101 and n > 85:
    r = 'A'
elif n < 86 and n > 60:
    r = 'B'
elif n < 61 and n > 35:
    r = 'C'
elif n < 36 and n > 0:
    r = 'D'
else:
    r = 'E'
    
print(r)

File: javascript/2344.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = parseInt(lines.shift()), r = ''

if (n < 101 && n > 85)     r = 'A';
else if (n < 86 && n > 60) r = 'B';
else if (n < 61 && n > 35) r = 'C';
else if (n < 36 && n > 0)  r = 'D';
else                       r = 'E';

console.log(r)